### PR TITLE
fix: Drop squash from testing

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by wtrocki** [comment]
Supplement to #276

File: packages/sync/test/LocalDirectiveFilterLink.test.ts [hunk 1]
```diff
@@ -26,11 +25,4 @@ describe("LocalDirectives", () => {
     expect(testLink.operations.length).equal(1);
     expect(!(hasDirectives(["onlineOnly"], testLink.operations[0].query)));
   });
-
-  it("ensures multiple directives do not exist after local link", () => {
```
Comment (StephenCoady) [inline packages/sync/test/LocalDirectiveFilterLink.test.ts]: @wtrocki I left this one as it checks whether or not it works for mutliple directives - probably a good test anyway?

File: packages/sync/test/operations.ts [hunk 2]
```diff
@@ -113,93 +71,7 @@ export const requestWithOnlineDirective: GraphQLRequest = {
...
-            arguments: []
-          }]
-        }]
-      }
-    }]
-  },
-  extensions: {} as any
-};
 
-export const requestWithBothDirectives: GraphQLRequest = {
```
Comment (StephenCoady) [inline packages/sync/test/operations.ts]: @wtrocki depending on my other comment and whether you agree to leave it this could stay too

**Comment by wtrocki** [comment]
@StephenCoady I'm leaving that with you. Closing PR as it seems that this was a valid use case.

**Comment by StephenCoady** [comment]
some of the changes could stay, i.e. renaming. I pushed them to my branch there anyway.

**Pull request closed without merging**